CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.22.0-rc.0 / 2020-10-07
+
+* [CHANGE] web: Remove APIv2. #7935
+* [FEATURE] React UI: Implement missing TSDB head stats section. #7876
+* [ENHANCEMENT] UI: Clarify alert state toggle via checkbox icon. #7936
+* [ENHANCEMENT] Add tsdb startup duration metric. #7737
+* [ENHANCEMENT] Include GOOS and GOARCH in version. #7968
+* [ENHANCEMENT] Add a rule_group_samples metric. #7977
+* [ENHANCEMENT] Eureka SD: Add UserAgent header to Eureka client. #8006
```

Comment:
This is so minor that I wouldn't mention it, it has no user impact.

Reply:
Removed